modules/cli/src/main/scala/scala/cli/internal/ProcUtil.scala

```diff
@@ -43,4 +45,15 @@ object ProcUtil {
     new String(data, StandardCharsets.UTF_8)
   }
 
+  def interruptProcess(process: Process): Unit = {
+    def interrupt(pid: Long) =
+      if (Properties.isWin)
+        os.proc("taskkill", "/PID", pid).call()
+      else
+        os.proc("kill", "-2", pid).call()
+
+    val pid = process.pid()
+    interrupt(pid)
```

Comment:
NIT No need of the local method:
```suggestion
    val pid = process.pid()
    if (Properties.isWin)
      os.proc("taskkill", "/PID", pid).call()
    else
      os.proc("kill", "-2", pid).call()
```